docs/user_guides/primitives_pipelines/primitives/TadGAN.rst

```diff
@@ -51,11 +48,11 @@ argument                    type                description
 
     X = np.array([1] * 100).reshape(1, -1, 1)
     primitive = load_primitive('orion.primitives.tadgan.TadGAN', 
-        arguments={"X": X, "epochs": 5, "batch_size": 1,
+        arguments={"X": X, "y":X, "epochs": 5, "batch_size": 1,
                    "iterations_critic": 1})
 
     primitive.fit()
-    y, critic = primitive.produce(X=X)
+    y, critic = primitive.produce(X=X, y=X)
```

Comment:
Is there any necessary reason to expose y for the reconstruction-based method?

Reply:
Yep, sometimes we do not reconstruct all of `X` but only a selected subset of channels. In this example, we are using a univariate signal, that's why they are the same. 

I prefer to expose them rather than let the user specify the dim, because it makes it easier to retrieve the signal for debugging or visualization purposes.